contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -71,6 +71,7 @@ type T struct {
 	podStatus    api.PodStatus
 	prototype    *mesos.ExecutorInfo // readonly
 	allowedRoles []string            // roles under which pods are allowed to be launched
+	defaultRoles []string            // roles under which pods are scheduled if none are specified in labels
```

Comment:
let's use the same identifier for this through-out the whole code. defaultRoles or defaultPodRoles sound fine.